src/modal/modal.js

```diff
@@ -186,14 +197,22 @@ class Modal extends React.Component<ModalPropsT, ModalStateT> {
   };
 
   getSharedProps(): $Diff<SharedStylePropsArgT, {children?: React.Node}> {
-    const {animate, isOpen, size, role, closeable} = this.props;
+    const {
+      animate,
+      isOpen,
+      size,
+      role,
+      closeable,
+      unstable_ModalBackdropScroll,
+    } = this.props;
     return {
       $animate: animate,
       $isVisible: this.state.isVisible,
       $isOpen: !!isOpen,
       $size: size,
       $role: role,
       $closeable: !!closeable,
+      $unstable_ModalBackdropScroll: unstable_ModalBackdropScroll,
```

Comment:
Do you mind adding a dev warning in `componentDidMount` saying that this will be removed? Hard to find that information without reading the `types.js` file